test/unit/connection/test-connection-ssl-max-tls1.0-reject.js

```diff
@@ -0,0 +1,41 @@
+var assert     = require('assert');
+var common     = require('../../common');
+var connection = common.createConnection({
+  port : common.fakeServerPort,
+  ssl  : {
+    ca         : common.getSSLConfig().ca,
+    maxVersion : 'TLSv1'
+  }
+});
+
+var server = common.createFakeServer({
+  ssl: {
+    minVersion: 'TLSv1.2'
+  }
+});
+
+server.listen(common.fakeServerPort, function(err) {
+  if (err) throw err;
+
+  connection.ping(function(err) {
+    var NODE_MAJOR_VERSION = parseInt(process.versions.node.split('.')[0], 10);
+    if (NODE_MAJOR_VERSION >= 10) {
+      if (!err) assert.fail('Expected to fail due to mismatched TLS versions for Node12+');
```

Comment:
I think the error message should be Node10+ (or the check should be fixed to match the error message).